spec/amp-html-layout.md

```diff
@@ -160,6 +160,26 @@ In the following example, if the viewport is wider than `320px`, the image will
 </amp-img>
 ```
 
+### `disable-inline-width`
+
+All AMP elements that support the `responsive` layout, also support the `disable-inline-width` attribute. The presence of this attribute describes how the `sizes` attribute is used. When pairing `disable-inline-width` with `sizes`, the AMP element will propagate the value of `sizes` to the element's underlying tag, as with the `img` nested inside an `amp-img`, **without** getting the sizing behavior of `sizes` in AMP (which adds a width based on which media query matches).
```

Comment:
I think this needs to start by saying `sizes` (without `disable-inline-width`) will set an inline `width` style on the element.